### ISSUE TITLE
BUG: u.startsWith is not a function

### ISSUE THREAD

**Issue by Tazi0** [SukkaW/nolyfill#80]
```
➜ npx nolyfill install 
Found 58 redundant packages:
 array-buffer-byte-length 1.0.1
 array-includes 3.1.8
 array.prototype.flat 1.3.2
 array.prototype.flatmap 1.3.2
 arraybuffer.prototype.slice 1.0.3
 assert 2.1.0
 available-typed-arrays 1.0.7
 deep-equal 2.2.3
 define-properties 1.2.1
 es-set-tostringtag 2.0.3
 function-bind 1.1.2
 function.prototype.name 1.1.6
 get-symbol-description 1.0.2
 globalthis 1.0.4
 gopd 1.0.1
 has-property-descriptors 1.0.2
 has-proto 1.0.3
 has-symbols 1.0.3
 hasown 2.0.2
 internal-slot 1.0.7
 is-arguments 1.1.1
 is-array-buffer 3.0.4
 is-core-module 2.14.0
 is-date-object 1.0.5
 is-generator-function 1.0.10
 is-regex 1.1.4
 is-shared-array-buffer 1.0.3
 is-string 1.0.7
 is-symbol 1.0.4
 is-typed-array 1.1.13
 is-weakref 1.0.2
 isarray 0.0.1
 object-keys 1.1.1
 object.assign 4.1.5
 object.entries 1.1.8
 object.fromentries 2.0.8
 object.getownpropertydescriptors 2.1.8
 object.values 1.2.0
 promise.allsettled 1.0.7
 regexp.prototype.flags 1.5.2
 safe-array-concat 1.1.2
 safe-regex-test 1.0.3
 set-function-length 1.2.2
 side-channel 1.0.6
 string.prototype.codepointat 0.2.1
 string.prototype.matchall 4.0.11
 string.prototype.padend 3.1.6
 string.prototype.padstart 3.1.6
 string.prototype.trim 1.2.9
 string.prototype.trimend 1.0.8
 string.prototype.trimstart 1.0.8
 typed-array-buffer 1.0.2
 typed-array-byte-length 1.0.1
 typed-array-byte-offset 1.0.2
 typed-array-length 1.0.6
 typedarray 0.0.6
 unbox-primitive 1.0.2
 which-typed-array 1.1.15

u.startsWith is not a function
```
I got this error? doesn't explain anything else :/

**Comment by nonzzz** [comment]
Can you share your local env?

**Comment by Tazi0** [comment]
@nonzzz what would you like to see specifically? What local environment?

**Comment by nonzzz** [comment]
Like `Nodejs` version and which package manager and etc.

**Comment by Martinspire** [comment]
Same issue here:
Node v20.13.1
Npm 10.8.1

```
Found 19 redundant packages:
 array-includes 3.1.8
 array.prototype.findlastindex 1.2.5
 array.prototype.flat 1.3.2
 array.prototype.flatmap 1.3.2
 deep-equal 1.1.2
 function-bind 1.1.2
 gopd 1.0.1
 has-proto 1.0.3
 has-symbols 1.0.3
 hasown 2.0.2
 is-core-module 2.13.1
 isarray 1.0.0
 object.fromentries 2.0.8
 object.groupby 1.0.3
 object.values 1.2.0
 set-function-length 1.2.2
 side-channel 1.0.6
 typedarray.prototype.slice 1.0.3
 which-typed-array 1.1.15

Trace: TypeError: u.startsWith is not a function
    at <local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\package-manager-B5oEznic.js:90:95
    at Array.reduce (<anonymous>)
    at transformOldOverrides (<local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\package-manager-B5oEznic.js:90:74)
    at Object.overridesPackageJson (<local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\package-manager-B5oEznic.js:90:668)
    at async t.<anonymous> (<local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\cli.js:28:156)
    at async t.parseAsync (<local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\cli.js:6:4119)
    at async <local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\cli.js:28:225
    at <local path>npm-cache\_npx\11dfa4defc9aa6e3\node_modules\nolyfill\dist\cli.js:28:292

```

**Comment by nonzzz** [comment]
Ok, let me check what's happened.

**Comment by leefreemanxyz** [comment]
Does your package.json already have an overrides field defined? I just tried running it on a project and also got the u.startsWith error (running the latest Node LTS), but after cutting my overrides field out, the nolyfill install worked correctly.

**Comment by Martinspire** [comment]
> Does your package.json already have an overrides field defined? I just tried running it on a project and also got the u.startsWith error (running the latest Node LTS), but after cutting my overrides field out, the nolyfill install worked correctly.

Ah yes, indeed, if I remove the overrides, it works. So it seems it has trouble merging them?

**Comment by jdalton** [comment]
Fixed in master branch.

**Comment by SukkaW** [comment]
> Fixed in master branch.

Yeah. It was fixed by #64 and #65 by @NamesMT. I am closing the issue now.
